app/src/androidTest/java/org/standardnotes/notes/StarterActivityTest.java

```diff
@@ -153,13 +132,22 @@ public void createNote() {
 
         appCompatEditText14.perform(scrollTo(), replaceText("body1a"), closeSoftKeyboard());
 
-        upButton.perform(click());
+        if (!TestHelper.isScreenSw600dpAndLandscape(mActivityTestRule.getActivity())) {
+            pressBack();
+        } else {
+            // Not sure why adding this delay works, not a good solution.
+            try {
+                Thread.sleep(500);
```

Comment:
I was thinking it might be some animation stuff, but I haven't dug deep into what makes this work.